_translator-files/po/zh_CN/1-index.po

```diff
@@ -57,21 +42,22 @@ msgid ""
 "[//]: # Markdown sections should not be indented, otherwise they are treated as code blocks.\n"
 msgstr ""
 "---\n"
-"title: \"Jamulus - 在线播放音乐。与朋友一起。免费的。\"\n"
+"title: \"Jamulus - 与朋友一起免费的在线音乐播放。\"\n"
```

Comment:
```suggestion
"title: \"Jamulus - 与朋友一起免费的在线演奏音乐。\"\n"
```

The term "Play" should be 演奏 instead of 播放. Other changes look fine.

I've updated the associated translation string from Weblate's side, might need to trigger an update to ensure the PR also gets updated.

Reply:
Ok. Pushed the latest changes from weblate. Thanks!